ethpm/utils/module_loading.py

```diff
@@ -0,0 +1,25 @@
+from importlib import import_module
+
+from typing import Any
+
+
+def import_string(dotted_path: str) -> Any:
```

Comment:
If that's a generic concept we reuse across multiple projects, maybe it deserves a place in `eth_utils`?

Reply:
(Not too relevant for this PR but maybe open up an issue to discuss and potentially move later)

Reply:
sounds like a plan! i know it's also in py-evm, not sure about any other projects?

Reply:
Me neither but seems useful in general.